Clarify README regarding open-source license (#59)
pine64/OpenPineBuds: README.md

# PineBuds SDK

This is just the SDK from the wiki (so far), with dockerfile setup to make building easier, and my reverse engineered bestool integrated.
The docker image will build bestool for you first, then drop you into the dev container for building and flashing the buds.

> (NOTE: Currently, the SDK is not licensed under a 'open source' license. We are working to resolve this issue, and will be reaching out to contributors and other parties soon. For now, consider this SDK as 'All Rights Reserved'.)

Reviewer: ```suggestion
NOTE: Currently, the SDK isn't licensed under an open source license. We're working to resolve this issue and will be reaching out to contributors and other parties soon. For now, consider this SDK as 'All Rights Reserved'.
```

The only objective suggestion is that it should be **an** open-source project. Use **an** if the next syllable starts with the sound of a vowel.

The rest of the suggestions are subjective, feel free to ignore them. :+1:

Author: @SethFalco My mistake. I've ignored the "'re" changes because I think that comes across as less formal than the original.

I've also added "/'shared source'", to more accurately explain the situation. Let me know if this resolves your review.
